evm_arithmetization/src/prover.rs

```diff
@@ -136,7 +131,7 @@ where
             registers_after: registers_data_after,
         };
 
-        let (traces, mut public_values, final_values) = timed!(
+        let (traces, mut public_values, _final_values) = timed!(
```

Comment:
I think we can even remove it as an output of `generate_traces`. I don't remember why it was added but it doesn't seem to be used anywhere anymore.